src/components/base/BaseButton.vue

```diff
@@ -9,8 +9,11 @@ export default defineComponent({
     ArrowLeftIcon,
   },
   props: {
-    // TODO: add below line after implementation of ROUTING
-    // to: { type: [Object, String], default: null, required: false },
+    to: {
+      type: [Object, String],
+      default: null,
+      required: false,
```

Comment:
If you provide a default value - you don't need to specify `required: false` 💡 
```suggestion
```

Reply:
I'm like 99% sure this is either an autocompletion (that I've never experienced before) or just the flow of coding made me put that in here, because I knew that `required: false` is a default in that scenario 😅 
Thank you for finding that! 🙏